helpers/compile/plugins/fill-plugin/fillers/decimal-small.ts

```diff
@@ -1,5 +1,3 @@
-import crypto from 'node:crypto'
```

Comment:
ℹ️ That's the main culprit that broke things. Added it in https://github.com/prisma/prisma/pull/26437/files based on editor/lint suggestions...